annotations/views/repository_views.py

```diff
@@ -73,7 +73,7 @@ def retrieve(self, request, pk=None, repository_pk=None):
         try:
             master_text = Text.objects.get(uri=result.get('uri'))
         except Text.DoesNotExist:
-            master_text = None
+            master_text = Text.objects.create(uri=result.get('uri'),title=result.get('name'),public=result.get('public'),content_type=result.get('content_types'),repository_source_id=result.get('id'),repository_id=repository_pk,addedBy_id=1)
```

Comment:
What is this doing? Is this from a different ticket? My intuition says you probably shouldn't be creating something when your trying to delete something.

Reply:
This was from ticket 311. It had to do with adding text. I made the same edit here as it was making it difficult to test the delete functionality without the add text button working.